src/shared/lib/resources/proposal/sprint-with-us.ts

```diff
@@ -56,18 +64,19 @@ export interface SWUProposal {
   history?: SWUProposalHistoryRecord[];
   submittedAt?: Date;
   opportunity: SWUOpportunitySlim;
-  organization: OrganizationSlim;
+  organization?: OrganizationSlim;
   inceptionPhase?: SWUProposalPhase;
   prototypePhase?: SWUProposalPhase;
-  implementationPhase: SWUProposalPhase;
-  references: SWUProposalReference[];
-  attachments: FileRecord[];
+  implementationPhase?: SWUProposalPhase;
```

Comment:
Could you remind me -- why did this become optional?

Reply:
For the anonymized proposals - we can't return any phase information as it would contain team members, which could be used to identify the proposal.